lib/worker/batcher.ex

```diff
@@ -383,6 +383,7 @@ defmodule BorsNG.Worker.Batcher do
     end
   end
 
+  @spec complete_batch(Status.state, Batch.t, [Status.t]) :: :ok
```

Comment:
I did not explore the code enought to find out if by design it is impossible for `waiting` status batch to hit the `maybe_complete_batch/1` fun; if there's no safety net there, then it's possible for this function to receive such state and that would raise a function argument error (we're only treating the cases of `ok`, `error` and `running`

Reply:
So, I will merge this PR now, but then, to improve documentation, I'll make the type

```
@type staged_state :: :ok | :error | :running
```

So we can specify it here, what do you think ? :^)